### PR TITLE
Use stable/victoria branches

### PR DESCRIPTION
**Pull request by priteau**

### CODE REVIEW

**Review by markgoddard — CHANGES_REQUESTED** [review]
Looks like we need to sync with kayobe-config @ stable/victoria. Master has cbe921e9ae1659091e4511643ba3f8fce883ffc4 and d4c3a30e54a0bcb30786fc67a6e0a311bf726fe8 which are probably almost backportable, but may need tweaks.

File: README.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ above and have already logged in (e.g. ``ssh centos@<ip>``).
    # Clone this Kayobe configuration.
    mkdir -p config/src
    cd config/src/
-   git clone https://github.com/stackhpc/a-universe-from-nothing.git kayobe-config -b master
+   git clone https://github.com/stackhpc/a-universe-from-nothing.git kayobe-config -b stable/victoria
 
    # Configure host networking (bridge, routes & firewall)
    ./kayobe-config/configure-local-networking.sh
```
[the remark this answers is not in the document]
Comment (priteau) [reply README.rst:108]: Thanks for the quick review.

**Comment by markgoddard** [comment]
Somehow we're still using CentOS 8.1 for the overcloud hosts, vs. 8.2 in stable/ussuri. Ideally we'd used 8.3. That change should go into master first an be backported though.

```
cirros_deploy_image_upstream_url: https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
```

**Review by markgoddard — APPROVED** [review]
LGTM. Diffs against master and stable/ussuri look sensible.

**Pull request merged**